ssl/quic/quic_tls.c

```diff
@@ -189,7 +190,7 @@ quic_new_record_layer(OSSL_LIB_CTX *libctx, const char *propq, int vers,
     return 1;
  err:
     *retrl = NULL;
-    OPENSSL_free(rl);
+    quic_free(rl);
```

Comment:
Why was this not picked up in other testing?
Calling OPENSSL_free should have resulted in a dummybio leak.

Reply:
> Why was this not picked up in other testing?

This only occurs on the error path. Most errors in this function are of the "cannot happen" variety. The fuzzer found one that could happen in the `yield_secret_cb` callback.